### ISSUE TITLE
what task does this work on

### ISSUE THREAD

**Issue by MPT109** [Ill-tableflip-U/Education-Perfect-Completer#3]
well i tried this on list task and other ones and my tab is completely unblocked but fsr it does not work even if i click alt/opt+r the task does not do anything

**Comment by Ill-tableflip-U** [comment]
![image](https://github.com/Ill-tableflip-U/Education-Perfect-Completer/assets/125164221/fd4dca91-86d5-4d8e-9ce8-57e37425df8e)
It works on EP lessons (as pictured above), If by list you mean what I think you mean then no it does not work on those.
However, you could make your own. I reverse engineered the way EP gets it's answers from the server as can be seen in https://github.com/Ill-tableflip-U/Education-Perfect-Completer/blob/main/get-questions.js
It should be able to scrape the answers for list tasks using this script I think... However I won't support lists in this project, I don't have the time.

**Comment by MPT109** [comment]
Does this work on mac? because option s doesnt work

**Comment by MPT109** [comment]
and ur supposed to reset inside of the task right?

**Comment by Ill-tableflip-U** [comment]
yeah reload the page once you get into the task. are you getting the popup when you load the page that says EP completer injected?

**Comment by MPT109** [comment]
yes but alt+s does nothing

**Comment by Ill-tableflip-U** [comment]
please send over your console output and I'll take a look. Please also send a screenshot of the page you are on when you reload and the message appears

**Comment by MPT109** [comment]
k

**Comment by MPT109** [comment]
is this supposed to be like this?
![Uploading Screenshot 2024-05-02 at 12.27.48 pm.png…]()

**Comment by MPT109** [comment]
<img width="1440" alt="Screenshot 2024-05-02 at 12 27 48 pm" src="https://github.com/Ill-tableflip-U/Education-Perfect-Completer/assets/167847598/1aacb915-36fe-473f-9e41-c7073eedc826">

<img width="565" alt="Screenshot 2024-05-02 at 12 30 39 pm" src="https://github.com/Ill-tableflip-U/Education-Perfect-Completer/assets/167847598/de53cf94-9c96-45eb-bee8-da8d633626ec">

**Comment by MPT109** [comment]
done

**Comment by Ill-tableflip-U** [comment]
is there anything in the console from injection.js?
Also are you reloading the page inside the task? you need to click start on the task and then reload the page (i will probably fix this in the future so no reloading is required)

**Comment by MPT109** [comment]
yes i went inside the task clicked reload and alt+s nothing happens and says nothing about injection.js

**Comment by MPT109** [comment]
man i don't know crap about html and js i only know like lua and c++

**Comment by Ill-tableflip-U** [comment]
does it say ep completer status: true in the popup once you do alt + s

**Comment by MPT109** [comment]
uhm no

**Comment by MPT109** [comment]
this is required to be on windows right?

**Comment by MPT109** [comment]
cuz i see something to check if ur on windows

**Comment by Ill-tableflip-U** [comment]
no that shouldnt affect it.... thats just sending the request to EP servers and pretending to be on a windows machine... ill test on my mac in a sec but can you try injecting it and doing alt s and then copy the console and paste it here (not a screenshot) also can you give me the URL of the EP task you are trying to run

**Comment by MPT109** [comment]
so i tried the thing on windows and it works but option+s does not get recognised and the status thing does not pop up

**Comment by Ill-tableflip-U** [comment]
so ep completer works fully on windows for you but not on mac, correct?

**Comment by MPT109** [comment]
correct

**Comment by MPT109** [comment]
bros probably scrolling through his javascript and dying rn

**Comment by Ill-tableflip-U** [comment]
got it, I know what the problem is. I have released an update so that it will work, I tried it on my own mac. yeah lol

**Comment by MPT109** [comment]
tysm

**Comment by MPT109** [comment]
if this works on list task we will earn so many points

**Comment by Ill-tableflip-U** [comment]
Nah. I've really moved onto other things myself. This was mainly to complete the stuff that my teachers set me, which is not lists. Still, I have the method for retrieving questions public if you ever wanted to do your own.

**Comment by MPT109** [comment]
yes pls
i think theres a simple script to fetch answers and insert it into ep
